README.md

```diff
@@ -105,7 +105,14 @@ BASE_DOMAIN=your.valid.domain.com
 CLUSTER_NAME=clustername
 # Set your valid DNS VIP, such as 1.1.1.1 for 'ns1.example.com'
 DNS_VIP="1.1.1.1"
+# Set your IP Network stack family, determines the IP address family of the cluster
+# subnet and service subnet, defaults to `v6`
+IP_STACK="v4"
+# Set your host IP Network stack family, determines the IP address family of the provisioning
+# network and external subnets, defaults to the value of `${IP_STACK}`
+HOST_IP_STACK="v4"
```

Comment:
I think there are some [unresolved issues](https://github.com/openshift-metal3/dev-scripts/pull/1226) with setting HOST_IP_STACK to non-default values so I'm wondering if we should mention it in the README?

Reply:
Thanks, done.